docs/examples/llamacpp/images/llamacpp-leader/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# syntax=docker/dockerfile:1
+FROM debian:bullseye AS builder
+
+RUN apt-get update && apt-get install --yes git cmake g++
+
+RUN mkdir /workspace
+WORKDIR /workspace
+RUN git clone https://github.com/ggerganov/llama.cpp.git
+
+WORKDIR /workspace/llama.cpp
+
+# GGML_RPC=ON: Builds RPC support
+# BUILD_SHARED_LIBS=OFF: Don't rely on shared libraries like libggml
+RUN cmake . -DGGML_RPC=ON -DBUILD_SHARED_LIBS=OFF -DGGML_DEBUG=1
+RUN cmake --build . --config Release --parallel 8
```

Comment:
is the parallel here "tensor parallelism" or "pipeline parallelism"?

Reply:
This is just running the cmake build in parallel.  Just a slightly faster docker build, no runtime effect :-)